engine/consensus/matching/core_test.go

```diff
@@ -363,13 +368,66 @@ func (ms *MatchingSuite) TestSealableResultsValid() {
 	valSubgrph := ms.ValidSubgraphFixture()
 	ms.AddSubgraphFixtureToMempools(valSubgrph)
 
+	// generate two receipts for result (from different ENs)
+	receipt1 := unittest.ExecutionReceiptFixture(unittest.WithResult(valSubgrph.Result))
+	receipt2 := unittest.ExecutionReceiptFixture(unittest.WithResult(valSubgrph.Result))
+	ms.ReceiptsDB.On("ByBlockIDAllExecutionReceipts", valSubgrph.Block.ID()).Return([]*flow.ExecutionReceipt{receipt1, receipt2}, nil)
```

Comment:
We don't have ByBlockIDAllExecutionReceipts anymore, this is out of sync with master or broken

Reply:
thanks. Fixed tests